packages/react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -98,6 +98,10 @@ class RCTAztecView: Aztec.TextView {
 
     // MARK: - Font
 
+    /// Flag to enable using the defaultFont in Aztec for specific blocks
+    /// Like the Preformatted and Heading blocks.
+    private var blockUseDefaultFont: Bool? = false
```

Comment:
Since the `blockUseDefaultFont` is always initialized to `false` we can make it non-optional and remove the [force unwrapping on line 712](https://github.com/WordPress/gutenberg/pull/37663/files#diff-3523b4a5a519e3975d67f8a202df9e8f91b3e3aff0168c86598c9e2abcbdf890R712)